plugins/parodos/src/hooks/useWorkflowDefinitionToJsonSchema/jsonSchemaFromWorkflowDefinition.ts

```diff
@@ -158,7 +154,7 @@ function transformWorkToStep(work: WorkType): Step {
       ...getJsonSchemaType(type),
     });
 
-    if (options.length > 0) {
+    if (options && options.length > 0) {
```

Comment:
```suggestion
    if (options.length > 0) {
```

Reply:
`options` is initialsed to `[]` on line 147, you don't need the extra check

Reply:
the options can probably be null if the parameter isn't 'Select'